File: 250317/시력 검사/eye-test.cpp
#include <iostream>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    double left_eye, right_eye = 0;
    cin >> left_eye >> right_eye;

    if(left_eye >= 1.0 && right_eye >= 1.0)
    {
        cout << "High";
    }
    else if(left_eye >= 0.5 && right_eye >= 0.5)
    {
        cout << "Middle";
    }
    else
    {
        cout << "Low";
    }
    return 0;
}

File: 250317/시력 검사/eye-test.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double left_eye = sc.nextDouble();
        double right_eye = sc.nextDouble();

        if (left_eye >= 1.0 && right_eye >= 1.0) {
            System.out.println("High");
        } else if (left_eye >= 0.5 && right_eye >= 0.5) {
            System.out.println("Middle");
        } else {
            System.out.println("Low");
        }
    }
}